CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ## v1227
 
 ### Added INP and long tasks reporting
-The interaction-to-next-paint metric is now calculated and reported at the end of user sessions, via [Google CWV](https://github.com/GoogleChrome/web-vitals) library. In addition, long continuously executed and blocking scripts detected by the PerformanceLongTaskTiming API is also forwarded to New Relic.
+The interaction-to-next-paint metric is now calculated and reported at the end of user sessions, via [Google CWV](https://github.com/GoogleChrome/web-vitals) library. In addition, long continuously executed and blocking scripts detected by the PerformanceLongTaskTiming API is also forwarded to New Relic. This functionality is `off` by default, until a curated UI experience is created to utilize this data.
```

Comment:
I would break this apart or more clearly indicate that the functionality that is off is the long tasks. This currently reads that it might be LT and INP.

Reply:
Suggesting a change in a comment below.